lib/bitcoinerlab.js

```diff
@@ -0,0 +1,144 @@
+const secp256k1 = require('@exodus/bitcoinerlab-secp256k1')
+let elliptic
```

Comment:
The point is to use `bitcoinerlab` since it's faster with Hermes

Reply:
Sure but why are we not loading the legacy library by default here, I see a lot of calls to load only if used, I wonder if it's such a perf impact to load elliptic by default rather than sprinking `loadLegacy` calls everwhere we need it